gap/matrix.gi

```diff
@@ -239,8 +239,9 @@ function(ri, G)
       else   # exactly two blocks:
           # FIXME: why don't we just compute a precise set of generators of the kernel?
           # That should be easily and efficiently possible at this point, no?
+          # The kernel is abelian, so we don't need to do normal closures.
+          findgensNmeth(ri).func := FindKernelRandom;
```

Comment:
Is "findgensNmeth(ri).func" correct? I have not found it at any other point in the package. Maybe "findgensNmeth(ri).method"?

Reply:
Well spotted. The documentation says that `findgensNmeth(ri)`  is a record with at least two components bound, namely `method` and `args`.

Reply:
Nice. That code must have been called by the tests I think. So the docs may be outdated. I'll double check when I'm back from holidays next week.

Reply:
Oh, or it simply never got called since I put it into the wrong component. ^^